website/docs/api-backends.md

```diff
@@ -1572,7 +1574,7 @@ with use_ops("cupy"):
 
 | Argument   | Type         | Description                                           |
 | ---------- | ------------ | ----------------------------------------------------- |
-| `ops`      | <tt>str</tt> | `"numpy"` or `"cupy"`.                                |
+| `ops`      | <tt>str</tt> | `"numpy"`, `"apple"`, `"cupy"` or `"mps"`.            |
```

Comment:
```suggestion
| `ops`      | <tt>str</tt> | `"numpy"`, `"apple"`, `"cupy"`, or `"mps"`.           |
```

Reply:
Ah, I thought we didn't do Oxford comma?

Reply:
Yea this suggestion was wrong, in the previous one the comma was missing between the second and third entity and my brain thought it was the same here, but it's not 😉

Reply:
Fixed the other one 😁